apps/api/src/writer.ts

```diff
@@ -26,10 +27,14 @@ export const handleSpaceDeployed: starknet.Writer = async ({ blockNumber, event,
 
   if (!event) return;
 
-  await instance.executeTemplate('Space', {
-    contract: event.contract_address,
-    start: blockNumber
-  });
+  const classHash = event.class_hash;
+
+  if (classHash === starknetNetworks.sn.Meta.masterSpace) {
+    await instance.executeTemplate('Space', {
+      contract: event.contract_address,
+      start: blockNumber
+    });
+  }
```

Comment:
This API is deployed for multiple networks, here we are only checking mainnet network `sn` so this won't work when indexing `sn-sep`.

The way we handle it is via `networkProperties` in `overrides.ts` files.

You just define certain property for example `masterSpace` and read it from `config.Meta.masterSpace` (here `config` will be network specific).

This way correct value will be used for all networks.

Reply:
b1d3a6e98d872233b40bc1356ae81c385ed5fe63

named it `spaceClassHash` but let me know if you prefer `masterSpace`